### ISSUE TITLE
Collaboration Cycle Kickoff [VA.gov Forms Strike Team, Form Still In Progress and/or Submitted Notifications]

### ISSUE THREAD

**Issue by megansiddle** [department-of-veterans-affairs/va.gov-team#37421]
## Steps to complete the Collaboration Cycle Kickoff

**VFS Product Manager**

- [x] Create this issue and notify the Collaboration Cycle team of this ticket in the in #vfs-platform-support ([see Slack Workflow (User guide](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)).  
- [x] Answer the questions below and attach any artifacts you have available.
- [x] Close this ticket when kickoff is complete and Platform has provided the recommended touchpoints. 

**Platform**

- [x] Review the responses and comment in ticket with recommended touchpoints and practice area involvement.
- [x] Platform: Add project to the [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).

## Artifacts (if available)

Provide links or documents for the following: 

- In Progress Reminders: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va.gov-strike-team/healthcare-in-progress-reminders.md
- Wireframes, mockups, or prototypes
- Any other artifacts you have so far 

## Questions
Please refer to the guidance in [Getting Started with the Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Getting-started.1760493650.html) for detailed information and examples for the below questions.

### 1. Will your work result in visible changes to the user experience?
- [x] Yes
- [ ] No

### 2. Is your work a new product or an iteration to an existing product?
- [x] New
- [ ] Iteration

### 3. If this is an iteration, please tell us what product or feature you're iterating.  

Product or feature: ____________

### 4. Are you making changes to the visual design?
- [ ] Yes
- [x] No

### 5. Are you doing research with VA.gov users?
- [x] Yes
- [ ] No

### 6. Will your work involve changes to: (select all that apply)

- [x] Tools and applications*
- [ ] Static pages
- [ ] Both

*For tools and applications, take into account if any static page entry points will need updates.*

### 7. Will your work include:  

- [x] New content
- [ ] Changes to existing content

### 8. Are you making a change to the user flow?
- [x] Yes
- [ ] No

### 9. Are you making a change to urls, navigation, or entry points?
- [x] Yes
- [ ] No

### 10. Does your product or feature have Google Analytics tracking and a KPI dashboard in Domo?
- [x] Yes
- [ ] No

### 11. If yes, do you need to capture any other analytics or metrics in addition to what you're currently tracking?

- [ ] Yes
- [x] No

### 12. Is there a GitHub label in the va.gov-team repo for this product?

- [x] No. 
- [ ] Yes. Please specify: ____________

### 13. Who is the VA business owner for this body of work?

- [x] VHA (Veterans Health Administration)
- [x] VBA (Veterans Benefits Administration)
- [x] NCA (National Cemetery Administration)
- [x] VACO (Veterans Affairs Central Office)
- [ ] Other. Please specify: ____________

## Additional information
Please refer to the [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) guidance on Platform website for more information about the Collaboration Cycle.
FYI: @shiragoodman , @chaseakirby

**Comment by shiragoodman** [comment]
Hey @megansiddle  - are you implementing this with the Health Care Application form first? And based on your responses, it sounds like you'll be treating this notification as it's own product... not part of the Health Care Application product?

If easier, we can wait to discuss later today when we meet 1:1. Thanks!

**Comment by shiragoodman** [comment]
Hi @megansiddle. Thanks for completing this Collaboration Cycle Kickoff ticket! Based on your responses, your project is a new tool and/or application so we are recommending the following Collaboration Cycle touchpoints: 

- [Design Intent](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Design-intent.1782153235.html)
- [Research Plan Review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Research-plan-review.1781891143.html)
- [Analytics Request](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) -  I recommend you reach out to the Analytics & Insights team to see if this is necessary. You can ask in Slack - just follow the [Slack support guidance. ](https://depo-platform-documentation.scrollhelp.site/support/Getting-help-from-the-Platform-in-Slack.1439138197.html)
- [Midpoint Review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Midpoint-review.1781039167.html)
- [Contact Center Review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Contact-center-review.1782317061.html) - I recommend you reach out to the Contact Center team to see if this is necessary. You can ask them directly via the #vsp-contact-center-support Slack channel.
- [Staging Review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Staging-review.1810137181.html)
- [Privacy, Security, Infrastructure Readiness Review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Privacy-and-security-review.1782317101.html)

The following practice areas will be included throughout the Collaboration Cycle process: 

- Design
- Content
- Information Architecture (IA)
- Quality Assurance (QA)
- Accessibility
- Product
- Research

We understand that scope of work and/or solutions can change as you work through your design process. If you find that your scope and/or solution changes, please reach out so that we can update and/or modify your touchpoints appropriately. 

Your project has been added to the [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886). When you’re ready to schedule your recommended Collaboration Cycle touchpoints, please review the guidance outlined on the [Collaboration Cycle Touchpoints](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Touchpoints.1768423528.html) page.

@megansiddle one last thing - do you have a github label for your team and for your product? We will need one to tag your work appropriately. If you don't have one, my team can propose label(s). Please let me know!

**Comment by megansiddle** [comment]
@shiragoodman Based on our previous research findings and discussions/approval with our VAPO, we are going to incorporate those findings into our updated designs for review then launch MVP notification for future iterations. Is it possible to remove the Research touchpoint since we are leveraging the previous findings?

**Comment by shiragoodman** [comment]
@megansiddle yes, absolutely. The Research Plan Review is contingent upon your research so if you're not doing research, then no Research Plan Review. I'll update the Platform Collaboration Point Tracker too.
